float64 curvature
int32[] costs
float64 colission_R
int32[] level_inds
dwal_planner/Pose2D_32[] poses
